### DISCUSSION TITLE
Do you know how to make your CC’s grey?

### DISCUSSION THREAD

**Discussion by giscus** [SSWConsulting/SSW.Rules.Content#9548] [Rules Discussions]
# aa84a3ef-e356-4340-8f33-e5d27579de6b

Visually filter out non-direct emails by making CC's grey in your inbox to reduce noise and increase productivity.

https://www.ssw.com.au/rules/how-to-make-emails-you-are-cc-ed-grey/

**Comment by bradystroud** [comment]
I wish this was possible on the new outlook :(
